packages/electrode-archetype-react-app/arch-clap.js

```diff
@@ -619,7 +649,7 @@ Individual .babelrc files were generated for you in src/client and src/server
     "check-dev": ["lint", "test-dev"],
 
     clean: [".clean.dist", ".clean.lib", ".clean.prod", ".clean.etmp", ".clean.dll"],
-    ".clean.dist": () => shell.rm("-rf", "dist"),
+    ".clean.dist": () => shell.rm("-rf", ...shell.ls("-d", "dist*")),
```

Comment:
why not use `buildDistDirs`?

Reply:
remove all the `dist-X` bundles, even `buildDistDirs` are different for the next time

Reply:
you must only worry about the target being built.  removing all you will have multiple builds stepping on each other.